mobile/src/main/java/ch/epfl/sdp/mobile/ui/Graphics.kt

```diff
@@ -58,6 +58,12 @@ val PawniesIcons.Search
 val PawniesIcons.Check
   get() = Icons.Default.Check
 
+val PawniesIcons.Mic
+  get() = Icons.Default.Mic
+
+val PawniesIcons.MicOff
+  get() = Icons.Default.MicOff
+
```

Comment:
It's the same that `PawniesIcons.GameMicOn`, right ? Do we really need to duplicate this ?

Reply:
Yes indeed. Just forgot to remove it 👍